src/sql/workbench/browser/modelComponents/media/declarativeTable.css

```diff
@@ -26,3 +26,14 @@
 	width: 100%;
 	text-align: center;
 }
+
+.declarative-table .toggle-more {
+	background-size: 16px !important;
```

Comment:
What are you overriding here that requires the important?

Reply:
I don't actually need that, also as a good practice, I am adding more specific selectors.